apis/sgv2-restapi/src/main/java/io/stargate/sgv2/restapi/service/resources/CQLResource.java

```diff
@@ -36,7 +36,7 @@
 @SecurityRequirement(name = RestOpenApiConstants.SecuritySchemes.TOKEN)
 @Tag(ref = RestOpenApiConstants.Tags.DATA)
 @EndpointDisabled(
-    name = "stargate.rest.cql.disabled",
+    name = "stargate.rest.cql-disabled",
```

Comment:
Looks like Quarkus 3.7 started validating matching of config -- and structure in `RestApiConfig` did not match: it used "stargate.rest.cqlDisabled" (not "stargate.rest.cql.disabled") so system property needs to match wrt capitalization.